src/scrabble.js

```diff
@@ -1,5 +1,98 @@
-function scrabble() {
 
+// Creating a function called scrabble - assigning 'word' (can use any word) as a parameter(variable that the function will have access to when it is called). In function body after {} is where logic is displayed, therefore, using an if statement to that is equal to null and returns 0
+
+// According to w3schools - "\t" - meaning of this is tabs and "\n" - meaning of this is newline character - therefore, we want to return a score of "0" if tab or newline is entered.
+
+// Remove whitespace - returns 0 for white space - according to MDN, we can use the 'trim' method that removes the whitespace.
+// const greeting = '   Hello world!   ';
+// For e.g,
+// console.log(greeting);
+// Expected output: "   Hello world!   ";
+
+// console.log(greeting.trim());
+// Expected output: "Hello world!";
+
+// Creating an if statement - that if a word has a length of 0 then this should return a score of 0
+
+// If words are entered in capitals then need to assign the toUpperCase() method to string
+
+function scrabble(word) {
+  if (word === null && word === '\n' && word === '\t' && word.length === 0) {
+    return 0
+  }
+}
+word = word.trim()
+
+word = word.toUpperCase()
+
+// Created a const variable 'letterScore' - then assigned the scores to the characters of the alphabet as later on - when we enter any word, a score will be retrived in the terminal.
+
+// Created a for loop because it is easier than listing multiple times. Create a variable for the sum of score 'totalScore'. Can now assign letterScore (const variable below). So totalScore = totalScore + letterScore. Use Return to return a totalScore.
+
+
+const letterScore = {
+  A: 1,
+  E: 1,
+  I: 1,
+  O: 1,
+  U: 1,
+  L: 1,
+  N: 1,
+  R: 1,
+  S: 1,
+  T: 1,
+  D: 2,
+  G: 2,
+  B: 3,
+  C: 3,
+  M: 3,
+  P: 3,
+  F: 4,
+  H: 4,
+  V: 4,
+  W: 4,
+  Y: 4,
+  K: 5,
+  J: 8,
+  X: 8,
+  Q: 10,
+  Z: 10
+}
+let totalScore = 0
+for (let character of word) {
+  totalScore += letterScore[character] 
 }
+return totalScore;
+
```

Comment:
i think this should come first at the top of the code - sort of sets the scene and then other functions come below. its important to return the value of each letter so that the info is held and letter values can be added together.
Also, look up charAt() method